### PR TITLE
Standardize the use of ``from gym import spaces``

### PR DESCRIPTION
**Pull request by qgallouedec**
## Description

From some reason, the linter prefers that syntax.
I also add a little change to the issue template to 
- get info system one command line (to avoid copying the dict and the sdout), 
- to avoid unwanted markdown formating
- to avoid referring to another issue

Screenshot 2022-12-26 at 08 20 26

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

- [x] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [x] I have updated the changelog accordingly (**required**).
- [x] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [x] I have updated the documentation accordingly.
- [x] I have opened an associated PR on the [SB3-Contrib repository](https://github.com/Stable-Baselines-Team/stable-baselines3-contrib) (if necessary)
- [x] I have opened an associated PR on the [RL-Zoo3 repository](https://github.com/DLR-RM/rl-baselines3-zoo) (if necessary)
- [x] I have reformatted the code using `make format` (**required**)
- [x] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [x] I have ensured `make pytest` and `make type` both pass. (**required**)
- [x] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

File: stable_baselines3/common/utils.py [hunk 1]
```diff
@@ -505,7 +507,7 @@ def get_system_info(print_info: bool = True) -> Tuple[Dict[str, str], str]:
         and a formatted string.
     """
     env_info = {
-        "OS": f"{platform.platform()} {platform.version()}",
+        "OS": re.sub(r"#(\d)", r"# \1", f"{platform.platform()} {platform.version()}"),
```
Comment (araffin) [inline stable_baselines3/common/utils.py:510]: maybe add a comment?

File: stable_baselines3/common/envs/identity_env.py [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+from gym import spaces
 
 from stable_baselines3.common.type_aliases import GymObs, GymStepReturn
 
+T = TypeVar("T", int, np.ndarray)
 
-class IdentityEnv(Env):
-    def __init__(self, dim: Optional[int] = None, space: Optional[Space] = None, ep_length: int = 100):
+
+class IdentityEnv(gym.Env, Generic[T]):
```
Comment (araffin) [inline stable_baselines3/common/envs/identity_env.py:12]: can we still use it the same way as before or is that a breaking change?

**Review by araffin — COMMENTED** [review]
Thanks for the PR =)
only minor remarks.
please update the version number too and I guess we need a PR for SB3 contrib?

File: stable_baselines3/common/envs/identity_env.py (hunk 2 above)
Comment (qgallouedec) [reply stable_baselines3/common/envs/identity_env.py:12]: Yes. It's just for type checking.

Comment (qgallouedec) [reply stable_baselines3/common/envs/identity_env.py:12]: Yes. It's just for type checking.

Comment (qgallouedec) [reply stable_baselines3/common/envs/identity_env.py:12]: To explain a little more:

With the previous typing, it was allowed for an instance to have the following typings:

```python
def reset(self) -> int:
def step(self, action: int) -> Tuple[int, ..., ..., ...]:
```

```python
def reset(self) -> int:
def step(self, action: int) -> Tuple[np.ndarray, ..., ..., ...]:
```

```python
def reset(self) -> int:
def step(self, action: np.ndarray) -> Tuple[int, ..., ..., ...]:
```

```python
def reset(self) -> int:
def step(self, action: np.ndarray) -> Tuple[np.ndarray, ..., ..., ...]:
```

```python
def reset(self) -> np.ndarray:
def step(self, action: int) -> Tuple[int, ..., ..., ...]:
```

```python
def reset(self) -> np.ndarray:
def step(self, action: int) -> Tuple[np.ndarray, ..., ..., ...]:
```

```python
def reset(self) -> np.ndarray:
def step(self, action: np.ndarray) -> Tuple[int, ..., ..., ...]:
```

```python
def reset(self) -> np.ndarray:
def step(self, action: np.ndarray) -> Tuple[np.ndarray, ..., ..., ...]:
```

From now on, the allowed typings are:

```python
def reset(self) -> int:
def step(self, action: int) -> Tuple[int, ..., ..., ...]:
```

```python
def reset(self) -> np.ndarray:
def step(self, action: np.ndarray) -> Tuple[np.ndarray, ..., ..., ...]:
```

After the switch to gym v0.26, there will be a lot of change of this kind to introduce.

**Comment by qgallouedec** [comment]
> please update the version number too and I guess we need a PR for SB3 contrib?

Done, and similar PR opened on sb3-contrib and rl-zoo3. Ready to merge IMO.

File: stable_baselines3/common/envs/identity_env.py (hunk 2 above)
Comment (araffin) [reply stable_baselines3/common/envs/identity_env.py:12]: alright ;)

**Review by araffin — APPROVED** [review]

**Pull request merged**